pom.xml

```diff
@@ -521,7 +540,8 @@
                 <artifactId>maven-checkstyle-plugin</artifactId>
                 <version>${maven-checkstyle-plugin.version}</version>
                 <configuration>
-                    <configLocation>src/main/resources/dd_checks.xml</configLocation>
+                    <configLocation>src/main/resources/checkstyle_ci.xml</configLocation>
+                    <includeTestSourceDirectory>false</includeTestSourceDirectory>
```

Comment:
Should include includeTestSourceDirectory too

Reply:
Ok, got it.